fenix/app/src/androidTest/java/org/mozilla/fenix/ui/BrowsingErrorPagesTest.kt

```diff
@@ -112,4 +112,54 @@ class BrowsingErrorPagesTest {
             verifyPageContent("Example Domain")
         }
     }
+
+    @Test
+    fun addressNotFoundErrorMessageTest() {
+        val invalidUrl = "ww.example.com"
+        val validUrl = "www.example.com"
+
+        navigationToolbar {
+        }.enterURLAndEnterToBrowser(invalidUrl.toUri()) {
+            waitForPageToLoad()
+            verifyAddressNotFoundErrorMessage()
+            clickTryAgainErrorPageButton()
+            verifyAddressNotFoundErrorMessage()
+        }
+
+        navigationToolbar {
+        }.enterURLAndEnterToBrowser(validUrl.toUri()) {
+            waitForPageToLoad()
+            verifyPageContent("Example Domain")
+        }
+    }
+
+    @Test
+    fun noInternetConnectionErrorMessageTest() {
+        val firstWebsite = "https://mozilla-mobile.github.io/testapp/v2.0/loginForm.html"
+        val secondWebsite = "www.example.com"
+
+        navigationToolbar {
+        }.enterURLAndEnterToBrowser(firstWebsite.toUri()) {
+            waitForPageToLoad()
+            verifyPageContent("Login Form")
+        }
+
+        setNetworkEnabled(false)
+
+        browserScreen {
+        }.openTabDrawer {
+        }.openNewTab {
+        }.submitQuery(secondWebsite) {
+            waitForPageToLoad()
+            verifyNoInternetConnectionErrorMessage()
+        }
+
+        setNetworkEnabled(true)
+
+        browserScreen {
+            clickTryAgainErrorPageButton()
```

Comment:
Could you please replace this with clickPageObject(itemWithText("errorTryAgain"))?

Reply:
Done !